creation/web_base/glidein_startup.sh

```diff
@@ -21,6 +31,7 @@ fi
 global_args="$*"
 # GWMS_STARTUP_SCRIPT=$0
 GWMS_STARTUP_SCRIPT="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/$(basename "${BASH_SOURCE[0]}")"
+export GWMS_STARTUP_SCRIPT=${GWMS_STARTUP_SCRIPT}       # for glidein reinvocation
```

Comment:
Couldn't we declare and export `GWMS_STARTUP_SCRIPT` in one line?

Reply:
That is a good suggestion. The `export` statement was something I added as part of my changes and it did not occur to me that declaring and exporting could be done in one line.